packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -130,6 +170,24 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         __ResourceMetering_init();
```

Comment:
For simplicity of the deployment process, add a bool to the initialize function here so that it can be initialized to being paused. In the constructor, call initialize with true so that the impl is always paused, preventing user error